website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -1,193 +1,236 @@
 # Inline Editing
 
-Inline editing for a column can be enabled for the whole column or at the cell level based on a condition. You may also allow users to [update existing rows](#row-level-mode) of data, or [add new rows](#adding-new-rows) entirely.
+Inline editing provides a quick way to edit data on the Table widget without navigating to the row details or moving away from the current screen.
 
 <VideoEmbed host="youtube" videoId="eIecDfvSOsU" title="" caption=""/>
 
+To try this feature yourself, take a look at the [sample app for inline editing](https://app.appsmith.com/app/editable-table/row-save-62d8f8d0e1c2ed505a0557cc).
+
+## Properties
+
+| Property                                           | Type        | Definition                                                                                                                                                                                                                                                               | Code Snippet                   |
+| -------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------ |
+| [**Editable**](#editable)                 | Formatting  | Controls whether cells of the column are editable                                                             | NA                             |
+| [**Cell Wrapping**](#cell-wrapping)            | Formatting  | Controls how overflowing contents of the column are handled.<br/> **on** - Contents get wrapped to the next line.<br/> **off** - Contents are truncated with an ellipsis. | NA |
+| [**Min**](#min)                      | Validation  | Sets the minimum allowed value.                                                                               | NA                             |
```

Comment:
Can we start adding the column type this field will be visible for? Like min max will only be visible for number columns